Description: Removing Bundler Requirement
 Removed the "require 'bundler/setup' from test.
 .
 ruby-default-value-for (3.0.0.1-1) unstable; urgency=medium
 .
   * Initial release (Closes: #767631)
Author: Balasankar C <balasankarc@autistici.org>
Bug-Debian: https://bugs.debian.org/767631

---

--- ruby-default-value-for-3.0.0.1.orig/test.rb
+++ ruby-default-value-for-3.0.0.1/test.rb
@@ -18,7 +18,6 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-require 'bundler/setup'
 require 'minitest/autorun'
 require 'active_record'
 require 'active_support/dependencies'
